src/test/java/com/navfort/runners/CukesRunner.java

```diff
@@ -15,7 +15,7 @@
         features = "src/test/resources/features",
         glue = "com/navfort/step_definitions",
         dryRun = false,
-        tags = "",
+        tags = "@US163",
```

Comment:
Hello Andrew, one suggestion, is it possible to make the tags empty by default once everyone done your own execution? easy for other people

Reply:
You are absolutely right, I'm sorry that I missed that